### ISSUE TITLE
Can this handle compressed files？

### ISSUE THREAD

**Issue by shuaixinjie** [microfocus-idol/idol-oem-tutorials#7]
Can this  tutorial_stream.c demo handle compressed files, such as zip, rar, 7z

**Comment by brian-cowe-mf** [comment]
Yes—check out the tutorial section [Extracting sub files](https://github.com/opentext-idol/idol-oem-tutorials/blob/main/tutorials/keyview_filter/programming_c.md#extracting-sub-files).
